### ISSUE TITLE
Syntax error: "(" unexpected

### ISSUE THREAD

**Issue by tutRPi** [microsoft/vscode-java-pack#887]
I have installed VS Code on Raspberry Pi 4B with the Extension Pack for Java. I also set the `JAVA_HOME` environment variable. However, when I open VS Code, I get this errors (output -> Language Support for Java):

```
[Error - 11:33:20 PM] Connection to server is erroring. Shutting down server.
[Error - 11:33:21 PM] Connection to server got closed. Server will not be restarted.
/home/pi/.vscode/extensions/redhat.java-1.2.0/jre/17.0.1-macosx-aarch64/bin/java: 1: Syntax error: "(" unexpected
```

Also, the Java Build Status doesn't stop spinning. This makes it not possible to run any Java application in VS Code.

**Environment:**

- VS Code Version: 1.63.2
- Extension Version: v0.20.0
- JDK Type & Version: 
openjdk version "11.0.13" 2021-10-19
OpenJDK Runtime Environment (build 11.0.13+8-post-Raspbian-1deb11u1)
OpenJDK Server VM (build 11.0.13+8-post-Raspbian-1deb11u1, mixed mode)
- OS Type & Version: Raspbian GNU/Linux 11 (bullseye)

**Comment by testforstephen** [comment]
redhat.java 1.2.0 embeds a wrong JRE for Raspberry platform, which will be fixed in next version 1.3.0. The planned release date is the end of this Friday.

So far, the workaround for you is to manually remove the directory `/home/pi/.vscode/extensions/redhat.java-1.2.0/jre/17.0.1-macosx-aarch64`.

**Comment by tutRPi** [comment]
Thank you, I can confirm after deleting the folder and installing the extension "Lombok Annotations Support for VS Code", it starts now.

**Comment by testforstephen** [comment]
Close this issue because Language Support for Java(TM) by Red Hat has released 1.3.0 to fix the wrong JDK issue.
